src/XenCert/XenCertCommon.py

```diff
@@ -262,3 +265,28 @@ def DisplayUsage(storage_type = None):
 def printHelpItem(item):
     Print(" %s %-20s\t[%s] %s" % (item[5], item[0], item[4], item[1]))
     
+def printCommand(inputs):
+    hidden_type = 0
+    for input in inputs[:]:
```

Comment:
I think for loop is needless. you can call index() to find parameter for password:
argv_tmp = argv[:]
idx_i = argv_tmp.index('-i')
\# change argv_tmp[idx_i+1] with hidden password
...

return “ ”.join(argv_tmp)